static/bidder-info/conversant.yaml

```diff
@@ -13,6 +13,6 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&rurl={{.RedirectURL}}"
+    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&rurl={{.RedirectURL}}"
     userMacro: ""
```

Comment:
missing userMacro?

Reply:
it's not missing, we don't use the userMacro.  We append the user id to the end of the redirect url instead.

Reply:
Generally it is good practice to replace the macro with UID. Let me know if you plan to make this change.

Reply:
I'm not going to make this change right now.  But I'll see how we want to add it in the near future.